Example/Example/Views/Pages/AuthPage.xaml.cs

```diff
@@ -16,6 +17,7 @@ public AuthPage()
 
         public void OnAuthenticateClicked(object sender, EventArgs args)
         {
+            IAuthService service = ServiceFinder.Resolve<IAuthService>();
```

Comment:
I added this here just as an example of how to get the correct `IAuthService` implementation